index.js

```diff
@@ -1,12 +1,27 @@
 const express = require("express");
 const app = express();
+const mongoose = require("mongoose");
+
+app.use(express.urlencoded({ extended: true }));
+app.use(express.json());
+
+mongoose
+    .connect(`${process.env.MONGO}`, {
+        useNewUrlParser: true,
+        useUnifiedTopology: true,
+        useCreateIndex: true,
+    })
+    .then(() => console.log("Connected to mongodb"))
+    .catch((err) => console.log(err));
 
 const port = 3000;
 
-app.get('/', (req, res) => {
+app.get("/", (req, res) => {
     res.send("Hello World!");
-})
+});
+
+app.use("/api/users/", require("./routes/login"));
```

Comment:
Shouldn't `/api/users` be different from `/api/auth` ? I think login seems unambiguous with `/api/auth`. We should have only user related CRUD in `/api/users/`.

Reply:
What i thought was we should include password update in the same file, which are all related to user actions. So i think I we should rename it to User.js instead of login.js. What say?

Reply:
Okay, will do this.